addons/account_edi_ubl_cii/data/ubl_21_templates.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="ubl_21_InvoiceLineType" inherit_id="account_edi_ubl_cii.ubl_20_InvoiceLineType" primary="True">
+        <xpath expr="//*[local-name()='Item']" position="before">
+            <t t-if="invoice.move_type == 'out_refund'">
```

Comment:
instead, can you put an id on `t` and change its condition like:
`<t id="condition_allowange_charg" t-if="invoice.move_type == 'out_invoice'"`
and use
```
<xpath="t[id='condition_allowange_charge']" position="attributes">
    <attribute name="if">invoice.move_type in ('out_invoice', 'out_refund')</attribute>
</xpath>
```

Reply:
Not sure if I got you right, but in UBL 2.0 we have (only for invoices ! AllowanceCharge is not present for credit notes):
 1. AllowanceCharge
 2. TaxTotal
For UBL 2.1 we have the same for invoice, *but* for credit note, it is swapped:
 1. TaxTotal
 2. AllowanceCharge
So the logic is: 
* in the template UBL 2.0, if invoice -> put AllowanceCharge before TaxTotal
* in the template UBL 2.1, if credit note -> put AllowanceCharge after TaxTotal